The android job is failing on main branch

### 🐛 Describe the bug

We got failures on the [android job](https://app.circleci.com/pipelines/github/pytorch/vision/10460/workflows/c65a1101-d936-4f76-95fe-f3b13a41c914/jobs/794213):

```
2021-09-16T09:48:15.003+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] FAILURE: Build failed with an exception.
2021-09-16T09:48:15.003+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2021-09-16T09:48:15.003+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * What went wrong:
2021-09-16T09:48:15.003+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] Could not determine the dependencies of task ':test_app:lintVitalFrcnnMnetv3BaseLocalRelease'.
2021-09-16T09:48:15.003+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] > Could not resolve all artifacts for configuration ':test_app:frcnnMnetv3CameraLocalDebugCompileClasspath'.
2021-09-16T09:48:15.003+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]    > Could not download nativeloader-0.8.0.jar (com.facebook.soloader:nativeloader:0.8.0): Skipped due to earlier error
2021-09-16T09:48:15.003+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] 
2021-09-16T09:48:15.003+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter] * Try:
2021-09-16T09:48:15.003+0000 [ERROR] [org.gradle.internal.buildevents.BuildExceptionReporter]  Run with --scan to get full insights.
```

Closer inspection to the log shows that the first failure was actually:

```
2021-09-16T09:47:41.588+0000 [DEBUG] [org.apache.http.impl.execchain.MainClientExec] Connection discarded
2021-09-16T09:47:41.588+0000 [DEBUG] [org.apache.http.impl.conn.PoolingHttpClientConnectionManager] Connection released: [id: 51][route: {s}->https://jcenter.bintray.com:443][total kept alive: 0; route allocated: 1 of 20; total allocated: 1 of 20]
2021-09-16T09:47:41.588+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Completing Build operation 'Download https://jcenter.bintray.com/org/pytorch/pytorch_android_torchvision/1.10.0-SNAPSHOT/maven-metadata.xml'
2021-09-16T09:47:41.588+0000 [DEBUG] [org.gradle.internal.operations.DefaultBuildOperationRunner] Build operation 'Download https://jcenter.bintray.com/org/pytorch/pytorch_android_torchvision/1.10.0-SNAPSHOT/maven-metadata.xml' completed
2021-09-16T09:47:41.588+0000 [DEBUG] [org.gradle.api.internal.artifacts.ivyservice.ivyresolve.ErrorHandlingModuleComponentRepository$ErrorHandlingModuleComponentRepositoryAccess] Error while accessing remote repository BintrayJCenter. Waiting 2000ms before next retry. 1 retries left
org.gradle.internal.resolve.ModuleVersionResolveException: Could not resolve org.pytorch:pytorch_android_torchvision:1.10.0-SNAPSHOT.
Caused by: org.gradle.api.resources.ResourceException: Unable to load Maven meta-data from https://jcenter.bintray.com/org/pytorch/pytorch_android_torchvision/1.10.0-SNAPSHOT/maven-metadata.xml.
```

Which seems a connection problem with the repo. At the time of writing of this ticket, the [Jcenter](https://jcenter.bintray.com/org/pytorch/pytorch_android_torchvision/1.10.0-SNAPSHOT/maven-metadata.xml) URL returns `403 Forbidden`. [sonatype](https://oss.sonatype.org/content/repositories/snapshots/org/pytorch/pytorch_android/1.10.0-SNAPSHOT/maven-metadata.xml) returns 200.

### Versions

Latest master 9275cc6

cc @seemethere @IvanKobzarev

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -15,7 +15,7 @@ allprojects {
             androidSupportAppCompatV7Version = "28.0.0"
             fbjniJavaOnlyVersion = "0.0.3"
             soLoaderNativeLoaderVersion = "0.8.0"
-            pytorchAndroidVersion = "1.10.0-SNAPSHOT"
+            pytorchAndroidVersion = "1.11.0-SNAPSHOT"
         }
 
         repositories {

